.github/workflows/network.yml

```diff
@@ -0,0 +1,70 @@
+name: Network CI
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'lib/vdsm/network/**'
+      - 'lib/vdsm/common/network/**'
```

Comment:
Can network tests be affected when something in common/* changes?

Reply:
Well it might be, I don't mind including whole common there if that's ok with others.

Reply:
Sounds right to depend on common.